### PR TITLE
Eperez master

### PR DESCRIPTION
**Pull request by enriquepablo**
Add stats object to flask apps

### CODE REVIEW

File: src/eduid_common/api/app.py [hunk 1]
```diff
@@ -100,6 +101,13 @@ def eduid_init_app_no_db(name, config, app_class=AuthnApp):
     app = init_sentry(app)
     app.session_interface = SessionFactory(app.config)
 
+    stats_hosts = app.config.get('STATS_HOSTS', False)
+    if not stats_hosts:
+        app.statsd = NoOpStats()
+    else:
+        #TODO port
```
Comment (johanlundberg) [inline src/eduid_common/api/app.py]: I propose to add,
stats_port = app.config.get('STATS_PORT', 8125)

File: src/eduid_common/api/app.py [hunk 2]
```diff
@@ -100,6 +101,13 @@ def eduid_init_app_no_db(name, config, app_class=AuthnApp):
     app = init_sentry(app)
     app.session_interface = SessionFactory(app.config)
 
+    stats_hosts = app.config.get('STATS_HOSTS', False)
```
Comment (johanlundberg) [inline src/eduid_common/api/app.py]: I think we only support one stats host (correct me if I am wrong) so can we rename STATS_HOSTS to STATS_HOST?

File: src/eduid_common/api/app.py [hunk 3]
```diff
@@ -100,6 +101,13 @@ def eduid_init_app_no_db(name, config, app_class=AuthnApp):
...
+    if not stats_hosts:
+        app.statsd = NoOpStats()
+    else:
+        #TODO port
+        app.statsd = Statsd(host=stats_hosts, port=22)
```
Comment (johanlundberg) [inline src/eduid_common/api/app.py]: port=stats_port

**Comment by johanlundberg** [comment]
I think we should bump the version of eduid_common as the api changes.

**Comment by enriquepablo** [comment]
Bumped version from 0.2.2b2 to 0.2.2b3

**Pull request merged**